Blog Post - AEO (#16742)
PostHog/posthog.com: contents/blog/aeo-advice.mdx

---
title: "How to grow your AEO function (without losing your mind)"
date: 2026-05-06
author:
    - natalia-amorim
featuredImage: >-
    https://res.cloudinary.com/dmukukwp6/image/upload/gardening_hog_9baff096a6.png
featuredImageType: full
category: General
tags:
  - Marketing
  - Growth
---

I learned SEO the traditional way: by telling a client I knew how to do it, then immediately Googling “what is SEO.” I must have done something right, because not long after that I was hired to do it full-time.

> 7 years later, fate (and LinkedIn) led me back to a familiar spot: *Can you lead our AEO efforts?*

Reviewer: ```suggestion
Seven years later, fate (and LinkedIn) led me back to a familiar spot.

"Can you lead our AEO efforts?"
```

> *Of course I can*, I replied, with the confidence of someone already opening 17 tabs.

Reviewer: ```suggestion
"Of course I can," I replied, with the confidence of someone already opening 17 tabs.
```

> The difference was that this time, everyone else in the industry was figuring it out too. The concept of AEO (or AIO, AISEO, GEO… \{insert here preferred acronym\}) had only existed for a few months. We were all still furiously Googling under the table.

Reviewer: ```suggestion
The difference was that this time, everyone else in the industry was still figuring it out too. The concept of AEO (or AIO, AISEO, GEO... \[insert preferred acronym here\]) had only existed for a few months. We were all still furiously Googling under the table.
```

> A year later, I'm reporting back from the AEO trenches with some good news: it works. It's the fastest-growing, highest-converting channel at PostHog – and it's not close either.

Reviewer: ```suggestion
A year later, I'm reporting back from the AEO trenches with some good news: it works. It's the fastest-growing, highest-converting channel at PostHog – and it's not even close.
```

Here's everything I wish I'd known on day one.

> ## Why you should give a damn

Reviewer: suggestion = i would collapse some of the stuff in this section into the intro (current it has a bit of a double intro)

For most of the last decade, growth has, among other things, been a function of how many strangers Google could send your way. Now an increasing share of it also depends on how often, and how accurately, an LLM mentions your name when asked for a recommendation.

This isn't theoretical for us; we're watching this happen in our numbers, in real time.

> **PostHog's LLM-referred traffic has grown 41x in 23 months**, 7.4x year-over-year, and every single quarter since Q2 2024 has set a new record, with no signs of slowing down.

Reviewer: i'd put this + the "does it convert" + the screenshot rights next to "AEO is our fastest growing channel" to surface the numbers at the top and grab attention/immediately create credibility, and then the first section can just be 1. The basics

![AI chatbot traffic count](https://res.cloudinary.com/dmukukwp6/image/upload/AI_chatbot_traffic_count_v2_a71b557dda.jpg)

If you're a skeptic like me, you're probably about to ask: *"WeLL, BuT dOeS iT cOnvErT?"*

Yes, yes it does. Better than almost anything else we have. 

![LLM conversion rates](https://res.cloudinary.com/dmukukwp6/image/upload/LLM_conversion_rates_v2_94d6700d0c.jpg)

> I'm not telling you this to humble brag *(well, maybe a little)*. **I'm telling you because the inverse is also true.**

Reviewer: ```suggestion
I'm not telling you this to humble brag (well, maybe a little). I'm telling you because the inverse is also true.
```

PostHog has [11 products and counting](/products), and (sadly) not all of them are being recommended by LLMs at the same scale or rate. While most models know and recommend us for [product analytics](/product-analytics), [session replay](/session-replay), [error tracking](/error-tracking) and [feature flags](/feature-flags), fewer acknowledge we also offer [a data warehouse](/data-stack), [LLM observability](/llm-analytics), [logs](/logs), and a lot more.

This visibility gap has turned into an obvious handicap for our growth in certain verticals, and closing it is something we're actively working on.

> You should be too.

Reviewer: ```suggestion
You should be, too. Because if you're not in the conversation, you're not in the game.
```

> **If you're not in the conversation, you're not in the game.**

Reviewer: ```suggestion
```

So, how do you get into the conversation? Start with the boring stuff.

## 1. The basics are still the basics

Here's the least sexy but most obvious piece of advice: most of ["traditional SEO"](/newsletter/seo-for-startups) still works.

Good structure, clear writing, semantic HTML, internal linking, fast loading pages, canonical tags, answering the damn question instead of burying it under 400 words of *"in today's fast-paced digital landscape…"*. All of it still matters.

> **The takeaway here isn't "SEO is dead, long live AEO." It's that SEO and AEO are drinking from the same well, the difference is how they drink it.**

Reviewer: ```suggestion
The takeaway here isn't "SEO is dead, long live AEO." It's that SEO and AEO are drinking from the same well, the difference is how they drink it.
```

Which means the "AEO checklist" most of the industry has already agreed on (the things I'm going to treat as settled for the purposes of this post) looks something like:

- **Make your content scannable.** Subheadings, short paragraphs, lists where lists belong.
- **Use natural language.** Write like a person explaining something to another person.
- **Cut the fluff.** If a sentence doesn't add information, delete it.
- **Answer questions directly.** If someone asks "what is X," tell them in the first sentence. Supporting context comes after.
- **Structure for retrieval.** H2s that are actual questions, definitions that can stand alone, summary tables, clear data, FAQs.
- **Be specific.** Models favor content with concrete details (real numbers, named tools, specific scenarios) over content that gestures vaguely at the same idea. **"Most teams hate GA4"** is forgettable; **"73% of teams using GA4 fantasize about leaving"** is citable. *(I made that up. Google legal team, don't @ me. It's probably true anyway.)*

For a deeper look at how we approach all of this in practice, [check out our AEO/SEO writing guide](/handbook/content/seo-guide).

> I'm not inventing the wheel here; everything I just listed would also be considered good SEO advice in 2019. The stakes are just higher now, and we have the benefit of hindsight to back it up.

Reviewer: ```suggestion
I'm not going to pretend I'm reinventing the wheel over here; everything I just listed would also be considered good SEO advice in 2019. The stakes are just higher now, and we have the benefit of hindsight to back it up.
```

> Anyone who's watched SEO mature over the last few years has already seen how high the upside can be when you take it seriously, and AEO is an amplifier of it (a big one at that). 

Reviewer: ```suggestion
Anyone who's watched SEO mature over the last few years has already seen how high the upside can be when you take it seriously. AEO takes the same fundamentals and amplifies them (by a lot).
```

tightened a bit to keep it from straying from the core message of this section ("back to the basics but on steroids")

> Don't sit this one out, and start by nailing the fundamentals.

Reviewer: ```suggestion
So given that it's almost the same game as before, there's no excuse to be sitting this one out. Nail the fundamentals and you're in.
```

tightening again to keep it close to the section's main arg

Author: I'm hesitant to say 'you're in' because realistically someone doing basic SEO stuff doesn't really guarantee they'll succeed in AEO, it just gets them to a basically where they _could, if that makes sense

Reviewer: valid, you can remove that! my main observation here was that the section was straying from "SEO and AEO are actually quite similar" into "AEO is an amplifier" at the very end.

the part i bolded below = still on track, after that it started to venture into the new argument. not a dealbreaker but it distracts before starting the next section

> **I'm not going to pretend I'm reinventing the wheel over here; everything I just listed would also be considered good SEO advice in 2019.** The stakes are just higher now, and we have the benefit of hindsight to back it up.
>
> Anyone who's watched SEO mature over the last few years has already seen how high the upside can be when you take it seriously, and AEO is an amplifier of it (a big one at that). 
> 
> Don't sit this one out, and start by nailing the fundamentals.

Author: agreed, will reframe this ending a bit!

> ## 2. Control what you can

Reviewer: "control what you can"/"clean your house first" sounds quite similar to "1. the basics"

the takeaway i got from section 2 was more about "don't get too overwhelmed by side quests/bonus tasks/the hype"

Author: Switched the focus a bit

If "the basics" above sound underwhelming, you're not alone.

Most people read a list like that and immediately wonder what else is there to do – and lucky for them, AEO has no shortage of side quests to embark on.

> > *Optimize your Reddit presence. Claim your Wikipedia page. Build a YouTube strategy. Get mentioned in traditional press. Chase down that one blog post a guy wrote on Medium in 2017 that's outranking your product page. Seed Quora. Pitch G2. Ghost-edit Hacker News comments (kidding) (but am I).*

Reviewer: ```suggestion
"Optimize your Reddit presence. Claim your Wikipedia page. Build a YouTube strategy. Get mentioned in traditional press. Chase down that one blog post a guy wrote on Medium in 2017 that's outranking your product page. Seed Quora. Pitch G2."
```

(hacker news manipulation, even if in jest, is a sensitive topic so i would avoid the joke here to be safe)

There's this underlying pressure to show up in every channel, every surface, every platform that might influence how LLMs talk about you – and it's true, they all might. That's the catch.

> **In a world where you don't really know what's feeding the models, everything is a potential signal, which means everything becomes a candidate for your time.**

Reviewer: ```suggestion
In a world where you don't really know what's feeding the models, everything is a potential input, which means everything becomes a candidate for your time.
```

small nit for tone, LLMs overuse the word "signals" haha

> Here's the rule I've settled on: **clean your house first.** Make sure your owned content is genuinely great. Everything else comes after.

Reviewer: ```suggestion
Here's the rule I've settled on: clean your house first. Make sure your owned content is genuinely great. Everything else comes after.
```

> That doesn't mean ignore the extra stuff forever; it means **do them when you have the capacity to do them well**, not when you're panicking that your competitor has a Reddit thread and you don't.

Reviewer: ```suggestion
That doesn't mean ignore the extra stuff forever; it means do them when you have the capacity to do them well, not when you're panicking that your competitor has a Reddit thread and you don't.
```

Full-ass one thing instead of half-assing twenty.

> ## 3. You don't need a Ferrari to learn how to drive

Reviewer: the metaphor works really well, but as a heading – if i were just skimming the outline – i might miss the actual message about AEO tooling. but i also see this as a valid style/tone choice, it's fun!

Author: Will switch this one too!

An important step of "doing AEO" is choosing your tools, and boy, that market got hot *fast*. If I never see another 23-slide deck demo, it'll be too soon.

> **My suggestion: buy for the problem you currently have, not the one you imagine having.**

Reviewer: ```suggestion
My suggestion: buy for the problem you currently have, not the one you imagine having.
```

If you're just starting out, pick something simple. Go for a tool that lets you learn the shape of the problem before you invest in solving it at scale. You don't need every fancy feature under the sun, you need ones you'll actually use.

*Want your agent pulling data? Look for a good MCP.*
*Live in spreadsheets? Make sure it exports cleanly.*

> **Honestly though, most tools out there are pretty similar. There's no clear winner, no obviously bad choice, and anyone telling you otherwise is selling something.**

Reviewer: ```suggestion
Honestly though, most tools out there are pretty similar. There's no clear winner, no obviously bad choice, and anyone telling you otherwise is selling something.
```

I went with [Gauge](https://www.withgauge.com/) for one reason that mattered to me more than anything else: their team builds *with* us, not *at* us.

When you're early in your journey and still figuring out what you need like I was, a team that actually ships your feature requests (instead of lobbing them into a ticket backlog where dreams go to die) beats a tool with fifty shiny functions you didn't ask for (or don't need yet). I wanted to be seen as a partner, not just an account number on a spreadsheet, and found that here.

*This is not sponsored. Although, Gauge, if you're reading this and feel like hooking me up with extra prompts, I won't say no.*

## 4. Call the bullshit, especially your own

There has never been an easier time to be a charlatan in this industry. It is trivially easy to say *"I increased prompt visibility 50x"* when you're the one who chose the prompts – and god knows if anyone actually asked them.

*"Who is the best Brazilian-Canadian content marketer with copper hair and a birthmark on her left hip?"* OH MY GOD IT'S ME!

I optimized the result to be me. Groundbreaking AEO work.

See what I did there?

> **In a world with no real prompt volume data (and no, don't come at me with tales of clickstream[^1]), there are no definitive results either.**

Reviewer: ```suggestion
In a world with no real prompt volume data (and no, don't come at me with tales of clickstream[^1]), there are no definitive results either.
```

![Prompt volume data](https://res.cloudinary.com/dmukukwp6/image/upload/pasted_image_2026_05_05_T21_18_36_079_Z_cd4186b7e3.png)

The whole discipline is mostly propped up on assumptions and operating on vibes. Which is fine – we're early, that's part of the process – but it means your job as someone actually doing the work is to be the most honest person in every room you're in.

> That means **questioning everything**, especially the stuff that makes you look good. When a number moves, ask why and don't accept the first explanation, even if it's yours.

Reviewer: ```suggestion
That means questioning *everything*, especially the stuff that makes you look good. When a number moves, ask why and don't accept the first explanation, even if it's yours.
```

People respect it, and more importantly, it keeps you from drinking your own Kool-Aid and building a strategy on top of a lie you told yourself.

[^1]: "Clickstream" is the catch-all term for behavioral data scraped (often without users realizing) from browser extensions, ISPs, or app SDKs. Vendors who sell AEO tools love it because it's the only data source that even pretends to estimate prompt volume. The problem: the panels can be limited (relative to the user populations they're trying to model), geographically skewed (mostly US, mostly Chrome users), and demographically self-selected. Extrapolating "global ChatGPT prompt volume" from a sample of a few million people who installed an extension that promised free coupons is, in my opinion, shaky at best, so tread lightly when using it as a source of truth.

## 5. You can just… ask

I'm not great at the whole not knowing thing, so I worked around it. We added a conditional question on our onboarding flow: whenever someone says they heard about us through AI, we ask them which prompt they used.

To my surprise, people answer. In detail. Sometimes long, long paragraphs of detail.

> **Now I have first-party data on the prompts actually leading people to us.** Real sentences typed by real humans with real credit cards.

Reviewer: ```suggestion
Now we have first-party data on the prompts actually leading people to us. Real sentences typed by real humans with real credit cards.
```

> As of today, **6,618** of them to be exact (and we've only been doing this for 3 months).

Reviewer: ```suggestion
As of today, 6,618 of them to be exact (and we've only been doing this for 3 months).
```

![LLM Conditional question](https://res.cloudinary.com/dmukukwp6/video/upload/LLM_conditional_question_video_04b5f21be7.mp4)

Every morning, our PostHog Slack automation drops a report with 50–100+ real prompts submitted by users who signed up on the previous day. [PostHog AI](/ai) helps me identify patterns and pull out useful little nuggets of information, which I feed straight back into our strategy. 

> It's an amazing feedback loop. The best part? It cost roughly zero dollars to set up, and it's become one of the most (if not the most) valuable strategic inputs in my entire reporting stack.

Reviewer: ```suggestion
It's an amazing feedback loop. The best part is that it cost roughly zero dollars to set up, and it's become one of the most (if not the most) valuable strategic inputs in my entire reporting stack.
```

> ![Prompt workflow](https://res.cloudinary.com/dmukukwp6/video/upload/LLM_Workflow_2_c45cf79570.mp4)

Author: Is this _too_ transparent? I am showing real data here, I'm comfortable with it but want to make sure this is kosher

Author: @andyvan-ph lmk what you think, I wanted to showcase PHAI but I understand if this video is pushing it a bit

We've watched hundreds of signups walk in asking for "an alternative to Google Analytics," dozens more for Sentry, Hotjar, Amplitude, Mixpanel. These reports have surfaced new content niches for us to focus on, specific framings of problems we hadn't thought to write about, adjacent tools and use cases to surface in our articles...

It's also a goldmine of unfiltered user thoughts, which I personally love. Some highlights:

![User responses](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/how_did_you_hear_about_us_3_81514f2058.jpg)

*Honorable mention to the user who told us they found out about PostHog in Fortnite voice chat, specifically talking to a Darth Vader NPC about analyzing the Fortnite economy. I think about him often.*

> ### Speaking of asking – talk to your peers.

Reviewer: this subsection would look neater as a callout box - otherwise adds another level of info hierarchy that isn't used elsewhere

Author: Agreed!

> Some of the best insights I've gotten came from just **asking other people doing this job what they're trying.**

Reviewer: ```suggestion
Some of the best insights I've gotten came from just asking other people doing this job what they're trying.
```

No single data source tells the full story. Not yet, anyway.

![Reporting stack](https://res.cloudinary.com/dmukukwp6/image/upload/aeo_reporting_stack_aa03395d44.jpg)

- **Referrer traffic** catches some of it, but not everything is properly tagged.
- **Self-reported attribution** catches more, but not everyone answers the question.
- **Bot analytics** (who's crawling you, how often, for what) helps you understand what the models are *seeing*, which is a different question than what the models are *saying*.
- **Prompt visibility tools** tell you what the models are saying, but only for the prompts you thought to track.

We can poke holes in every one of these sources.

Is it annoying? Yes, very. The last decade of clean(ish) attribution coddled us – anyone in growth, marketing, or analytics has gotten used to a dashboard where everything reconciled. AEO isn't going to give us that anytime soon.

If you're a founder watching your team try to report on this, give them grace; if you're the one reporting on it, godspeed.

So does that mean you don't report on it? No. Sometimes you tell the story even if all you have is duct tape and a dream.

> **The job is to connect the dots to the best of your ability and communicate that forward – and to be honest about which dots are real data and which ones are inference.**

Reviewer: ```suggestion
The job is to connect the dots to the best of your ability and communicate that forward – and to be honest about which dots are real data and which ones are inference.
```

## 7. Be prepared to start over

A few months in, I realized my prompt tracking strategy was partially wrong. We were tracking hundreds of AI-generated prompts that had no overlap with the prompts real people were actually using to find us. *(How did I figure that out? Refer to #4 and #5.)*

Which meant six months of historical data aged like milk.

When you're working in a discipline this new, any data feels sacred, so admitting to my team and to myself that ours needed to be scrapped was a tough pill to swallow. But we started over, and the new dataset is already more useful than the old one ever was.

It would've been easier if there was a playbook to defer to; there isn't.

Zoom out, though, and the same principle applies to everything else in AEO. The models are changing constantly; a feature that didn't exist last week can completely rewire the landscape you thought you'd figured out.

Anyone who lived through the Panda, Penguin, or Helpful Content updates in SEO (Google algorithm shifts that wiped out entire categories of content overnight) already knows this tune all too well: a win today is not a win tomorrow.

> **The strategies that got you here might get you nothing in six months; the key is not to fear it, it's to expect it.**

Reviewer: ```suggestion
The strategies that got you here might get you nothing in six months; the key is not to fear it, it's to expect it.
```

  - *PostHog example: "Open-source web analytics tool that won't sell my data."*
- **You ∩ Competitors: table stakes prompts.** What you and competitors both do. Everyone shows up here. Worth tracking as a baseline, but rarely worth fighting for as they're usually too broad, and nobody wins by being the fifth mention in a list of ten.
  - *Example: "Best web analytics tools."*
- **ICP ∩ Competitors: vulnerability prompts.** What your customer wants, competitors offer, you don't (or don't do well yet). You're losing these. Track them to learn where your ICP is being pulled instead.
  - *PostHog example: "Best web analytics tool with native Google Ads integration."*
- **All three: battleground prompts.** Your ICP wants it, you do it, your competitors do it too, which means the LLM has to pick someone from the shortlist when it answers. This is where the actual war happens. Optimize here first, hardest, most obsessively.
  - *PostHog example: "Best alternatives to google analytics for startups" or "best web analytics tool for developers"*

Layer that with intent (TOFU / MOFU / BOFU, aka where does it sit in the conversion funnel) and you have your priority list.

> **Battleground prompts at MOFU are where I spend most of my time.**

Reviewer: ```suggestion
Battleground prompts at MOFU are where I spend most of my time.
```

I'm not saying this from a moral high horse – we tried it too.

We tested one of the most popular and fancy ($$$) end-to-end AI content automation tools out there with the hopes we would be able to offload some of the content creation burden; you'd think throwing money at the problem would solve it, but after weeks of iteration, nothing it produced met our standards (or came close enough to make it worth the reputational risk and level of investment). So we're sticking to humans for now.

My current stance, which is roughly the sensible-person consensus:

**It would be silly not to use AI tools to speed up the work.** Research, briefs, outlines, drafting summaries, grammar edits, fact-checking against sources are all fair game to me and something we'll continue to invest and expand on.

**It would be sillier to think you can offload the whole thing.** Strategy, voice, taste, judgment about what to write at all, those should still be yours.

That's our take. Yours might be different.

Only one way to find out.

</details>

> ## The fine print

Reviewer: this might also make sense as a dropdown section? (it doesn't feel like fine print in terms of format right now)

Author: I'll rename this, I dont want to collapse it so I'll change the title

I didn't write this to claim I've figured out AEO. Far from it.

I wrote it because the version of me from a year ago would have loved to read it, and because the version of me a year from now will probably disagree with at least two points on this list.

> **Two things can be true at once: I might just be throwing pasta at the wall and seeing what sticks.**

Reviewer: ```suggestion
Two things can be true at once: I might just be throwing pasta at the wall and seeing what sticks.
```

> **…I might also be a really good pasta thrower.**

Reviewer: ```suggestion
I might also be a really good pasta thrower.
```
